frontend/server/k8s-helper.ts

```diff
@@ -127,3 +131,30 @@ export function getPodLogs(podName: string): Promise<string> {
       (error: any) => { throw new Error(JSON.stringify(error.body)); }
     );
 }
+
+export async function newTensorboardInstance(logdir: string): Promise<void> {
+  if (!k8sV1CustomObjectClient) {
+    throw new Error('Cannot access kubernetes Custom Object API');
+  }
+  // TODO: take the configuration below to a separate file
+  const group = 'kubeflow.org';
+  const version = 'v1beta1';
+  const namespace = 'kubeflow';
+  const plural = 'tensorboard';
+  const body = {
+    apiVersion: 'kubeflow.org/v1beta1',
+    kind: 'Viewer',
+    metadata: {
+      generateName: 'viewer-',
+      namespace: 'kubeflow',
```

Comment:
I do not think we should hard-code the namespace.

Reply:
When I used one-click deployment of kubeflow, I found all the resulting pods in "kubeflow" namespace. E.g., ml-pipeline-ui pod, ml-pipeline pod.  I guess the namespace is configurable via some yaml when deployment is performed? If that is the case, I can follow the same convention here if feasible.

Reply:
@IronPan How should we get the namespace properly?

Reply:
talked offline with @jingzhang36 for more context. it should not be hardcoded. 
Here contains the example how to customize the namespace
https://github.com/kubeflow/pipelines/tree/master/manifests